src/platforms/linux-openwrt.js

```diff
@@ -132,6 +132,39 @@ function uciCommit(key) {
   return proc.status === 0;
 }
 
+/**
+ * Get DHCP server status.
+ *
+ * @returns {boolean} Boolean indicating whether or not DHCP is enabled.
+ */
+function getDhcpStatus() {
```

Comment:
It's probably worth expanding the function name a bit. DHCP will run in 2 contexts: DHCP server for LAN side, and DHCP client for WAN side (or possiblly LAN side if doing an extender setup).

So I think that these should be called getDhcpServerStatus and setDhcpServerStatus

Something for future consideration: It's probably also worthwhile checking to see if we can stop the DHCP server process if it isn't being used. For my turris setup, I did:
```
/etc/init.d/odhcp disable
/etc/init.d/odhcp stop
```
to cause the DHCP server to not even run (which saves memory).

Reply:
I can rename, that's a good point.

According to the OpenWrt docs, odhcp is only used for IPv6, while dnsmasq is used for IPv4.

Reply:
So we probably want to stop/start/configure both then. I was following this guide: https://openwrt.org/docs/guide-user/network/wifi/dumbap and that does the dhcp.lan.ignore=1 to disable the IPv4 and also disabled odhcpd

Reply:
Done. Can you take a quick look?